shard-operator-parent/shard-operator-v1/src/main/java/com/redhat/service/smartevents/shard/operator/v1/resources/knative/KnativeBroker.java

```diff
@@ -1,4 +1,4 @@
-package com.redhat.service.smartevents.shard.operator.resources.knative;
+package com.redhat.service.smartevents.shard.operator.v1.resources.knative;
```

Comment:
Nitpick: we could keep `knative` POJOs in the `core` as it is going to be used also in `v2`

Reply:
Done. I've moved `knative` and `istio` to `-core`.